projects/1.18-fabric/assets/auto-workstations-fabric/autoworkstations/lang/zh_cn.json

```
@@ -0,0 +1,37 @@
+{
+  "itemGroup.autoworkstations.items": "自动化工作站",
+  "block.autoworkstations.iron_auto_crafting_table": "铁质自动工作台",
+  "block.autoworkstations.iron_auto_crafting_table.tooltip": "自动合成",
+  "block.autoworkstations.gold_auto_crafting_table": "金质自动工作台",
+  "block.autoworkstations.gold_auto_crafting_table.tooltip": "一个速度更快的自动工作台",
+  "block.autoworkstations.netherite_auto_crafting_table": "下界合金质自动工作台",
+  "block.autoworkstations.netherite_auto_crafting_table.tooltip": "可以一次性合成多达一组的物品",
+  "block.autoworkstations.iron_auto_furnace": "铁质自动熔炉",
+  "block.autoworkstations.iron_auto_furnace.tooltip": "可以将熔炼所获得的经验装瓶",
+  "block.autoworkstations.gold_auto_furnace": "金质自动熔炉",
+  "block.autoworkstations.gold_auto_furnace.tooltip": "一个速度更快的自动熔炉",
+  "block.autoworkstations.auto_enchanting_table_xp_inside": "不消耗经验的自动附魔台",
+  "block.autoworkstations.auto_enchanting_table_xp_inside.tooltip": "假装这个物品不存在",
+  "block.autoworkstations.iron_auto_enchanting_table": "铁质自动附魔台",
+  "block.autoworkstations.iron_auto_enchanting_table.tooltip": "用附魔之瓶来附魔物品，摆放书架以提升附魔等级",
+  "block.autoworkstations.gold_auto_enchanting_table": "金质自动附魔台",
+  "block.autoworkstations.gold_auto_enchanting_table.tooltip": "一个更快的自动附魔台",
+  "block.autoworkstations.iron_auto_experience_orb_vacuum": "铁质经验吸收器",
+  "block.autoworkstations.iron_auto_experience_orb_vacuum.tooltip": "吸取附近的经验球并将它们装瓶",
+  "block.autoworkstations.gold_auto_experience_orb_vacuum": "金质经验吸收器",
+  "block.autoworkstations.gold_auto_experience_orb_vacuum.tooltip": "具有比铁质经验吸收器更大的吸收范围",
+  "block.autoworkstations.iron_auto_anvil": "铁质自动铁砧",
+  "block.autoworkstations.iron_auto_anvil.tooltip": "可以用于自动融合两个物品或重命名物品",

```

```suggestion
  "block.autoworkstations.iron_auto_anvil.tooltip": "可以用于自动合并两个物品或重命名物品",
```
可见minecraft wiki铁砧页面